src/lib/clients/IOClients/apps/Builder.ts

```diff
@@ -105,15 +110,22 @@ export class Builder extends AppClient {
     app: string,
     zipFile: Buffer,
     stickyOptions: StickyOptions = { sticky: true },
-    params: RequestParams = {}
+    params: RequestParams = {},
+    requestHeaders: Record<string, any> = {}
   ) => {
-    return this.sendZipFile(routes.link(app), app, zipFile, stickyOptions, params)
+    return this.sendZipFile(routes.link(app), app, zipFile, stickyOptions, params, requestHeaders)
   }
 
-  public relinkApp = (app: string, changes: ChangeToSend[], params: RequestParams = {}) => {
+  public relinkApp = (
+    app: string,
+    changes: ChangeToSend[],
+    headersParams: Record<string, any>,
```

Comment:
`requestHeaders`, or just `headers`

Reply:
I think `relinkApp` and `linkApp`, instead of having `headers` as parameter they could have `linkId` as parameter, and then call:
```ts
return this.sendZipFile(routes.link(app), app, zipFile, stickyOptions, params, { [Headers.VTEX_LINK_ID]: linkID }
```
The difference in these approachs is semantic:
- Using headers: It seems a optional thing - the function user can pass any header it wants, not specifically the link-id header
- Using linkId as parameter: The function user *has* to provide a linkId on this operation - which is what we want to enforce